voicebox/connection.py

```diff
@@ -96,13 +99,20 @@ def send_message(self, message, msg_type=1):
             logging.debug("Connection reset while sending packet")
             self.kill(inform_client=False)
 
-    @staticmethod
-    def decrypt_packet(packet: bytes):
+    def decrypt_packet(self, packet: bytes):
+
+        for encryptor in self.encryption_pipeline:
```

Comment:
Consider adding error handling in the `decrypt_packet` and `encrypt_payload` methods to handle potential failures in the encryption process. This could be done using a try-except block around the encryption and decryption code. If an error occurs, it could be logged for debugging purposes. [important]